docs/shared/tutorials/react-standalone.md

```diff
@@ -25,273 +24,361 @@ Note, while you could easily use Nx together with your manually set up React app
 
-## Creating New Components
+Now let's change where the `build` results are output to in the `vite.config.ts` file.
 
-You can just create new React components as you normally would. However, Nx plugins usually also ship [generators](/features/generate-code). They allow you to easily scaffold code, configuration or entire projects. To see what capabilities the `@nx/react` plugin ships, run the following command and inspect the output:
+```{% fileName="vite.config.ts" highlightLines=["7-9"] %}
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
 
-```{% command="npx nx list @nx/react" path="myreactapp" %}
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
+  build: {
+    outDir: 'dist/react-app',
+  },
+});
+```
 
-NX   Capabilities in @nx/react:
+Now if you look at project details view again, you'll see that the `outputs` property for Nx's caching has been updated to stay in sync with the setting in the `vite.config.ts` file.
 
-   GENERATORS
+You can also add the `@nx/eslint` plugin to see how it infers `lint` tasks based on the ESLint configuration files.
 
-   init : Initialize the `@nrwl/react` plugin.
-   application : Create a React application.
-   library : Create a React library.
-   component : Create a React component.
-   redux : Create a Redux slice for a project.
-   storybook-configuration : Set up storybook for a React app or library.
-   component-story : Generate storybook story for a React component
-   stories : Create stories/specs for all components declared in an app or library.
-   component-cypress-spec : Create a Cypress spec for a UI component that has a story.
-   hook : Create a hook.
-   host : Generate a host react application
-   remote : Generate a remote react application
-   cypress-component-configuration : Setup Cypress component testing for a React project
-   component-test : Generate a Cypress component test for a React component
-   setup-tailwind : Set up Tailwind configuration for a project.
-   setup-ssr : Set up SSR configuration for a project.
-   federate-module : Federate a module.
+```text
+npx nx add @nx/eslint
+```
 
-   EXECUTORS/BUILDERS
+## Creating New Components
 
-   module-federation-dev-server : Serve a host or remote application.
-   module-federation-ssr-dev-server : Serve a host application along with it's known remotes.
+You can just create new React components as you normally would. However, Nx plugins also ship [generators](/features/generate-code). They allow you to easily scaffold code, configuration or entire projects. Let's add the `@nx/react` plugin to take advantage of the generators it provides.
+
+```text
+npx nx add @nx/react
+```
+
+To see what capabilities the `@nx/react` plugin ships, run the following command and inspect the output:
+
+```text {% command="npx nx list @nx/react" path="react-app" %}
+ NX   Capabilities in @nx/react:
+
+GENERATORS
+
+init : Initialize the `@nrwl/react` plugin.
```

Comment:
This should be `@nx/react` for the init generator. I'll open a PR to fix this in our `generator.json` files.